src/core/app/user.ts

```diff
@@ -1,25 +1,33 @@
 import { appConfig } from '~core/app_config';
 
-export type CurrentUser = {
-  id: string | null;
+export type UserProfileMain = {
```

Comment:
`UserProfileMain` is toooo abstract
UserProfileCredentials / UserProfileCreds may be?

Reply:
to be refactored in next iterations